scss/tech.scss

```diff
@@ -0,0 +1,43 @@
+.tech {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
why flex wrap?

Reply:
I dont know why I put but I removed it and everything goes ok